### PR TITLE
Add UUID to streaming OpenAI requests

### PR DESCRIPTION
**Pull request by SandersAaronD**
Untested

Fixes https://github.com/grafana/grafana/issues/75179

### CODE REVIEW

File: src/llms/openai.ts [hunk 1]
```diff
@@ -313,7 +313,7 @@ export function streamChatCompletions(request: ChatCompletionsRequest): Observab
   const channel: LiveChannelAddress = {
     scope: LiveChannelScope.Plugin,
     namespace: LLM_PLUGIN_ID,
-    path: OPENAI_CHAT_COMPLETIONS_PATH,
+    path: OPENAI_CHAT_COMPLETIONS_PATH + self.crypto.randomUUID(),
```
Comment (sd2k) [inline src/llms/openai.ts:316]: Just to keep the path a bit neater:

```suggestion
    path: OPENAI_CHAT_COMPLETIONS_PATH + '/' + self.crypto.randomUUID(),
```

**Review by sd2k — CHANGES_REQUESTED** [review]
Should be fine as long as users have upgraded their LLM app, but I'm not sure what to do about back-compat 🤔

Perhaps we need to add a `version` endpoint to the LLM app and store it somewhere as part of the `enabled` call. For older versions that endpoint will return a 404 so we know it's <0.3.0, and it'd work as expected for newer versions.

**Comment by SandersAaronD** [comment]
> Should be fine as long as users have upgraded their LLM app, but I'm not sure what to do about back-compat 🤔
> 
> Perhaps we need to add a `version` endpoint to the LLM app and store it somewhere as part of the `enabled` call. For older versions that endpoint will return a 404 so we know it's <0.3.0, and it'd work as expected for newer versions.

I like the new health endpoint; my only concern is that if we push release in order to ensure backwards-compatibility for this we might miss our window for the demo we're trying to do with the feature.

**Review by sd2k — APPROVED** [review]
Yeah I think that's fair enough, it's super early and marked as experimental so it's not unreasonable to expect users to upgrade regularly.

**Pull request merged**